src/app/account/components/tabs/connected-accounts/index.tsx

```
@@ -1,77 +1,35 @@
       </div>
-    );
-  }
-
-  return (
-    <div className="flex flex-col gap-2">
-      <p className="text-sm text-muted-foreground mb-2">
-        The following are previously connected accounts. Please manage all connections via your user profile going forward.
+      <p className="text-xs text-muted-foreground mt-4">
+        Note: Connections to external tools and services for use within workbenches
+        (like authorizing Google Calendar to access its data) are managed separately within
+        each workbench's settings.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)